gramps/gui/editors/editplace.py

```diff
@@ -176,6 +176,24 @@ def _setup_fields(self):
         #force validation now with initial entry
         self.top.get_object("lat_entry").validate(force=True)
 
+        self.latlon = MonitoredEntry(
+            self.top.get_object("latlon_entry"),
+            self.set_latlongitude, self.get_latlongitude,
+            self.db.readonly)
+
+    def set_latlongitude(self, value):
+        try:
+            coma = value.index(',')
+            self.longitude.set_text(value[coma+1:])
+            self.latitude.set_text(value[:coma])
+            self.top.get_object("lat_entry").validate(force=True)
+            self.top.get_object("lon_entry").validate(force=True)
+        except:
```

Comment:
I didn't want to reinvent the wheel. So I use what already exists.
You enter 1,2. Neither google, openstreetmap or other map services will give you these values.
It is written : **copy/paste**. You entered manually these values.
If you want to create a new widget, I don't like this : There are too many widgets in gramps. I think this is clear like that. I can add some controls if you want.